library/src/webapp/js/spinner.js

```diff
@@ -118,7 +118,10 @@ SPNR.disableControlsAndSpin = function( clickedElement, escapeList )
 {
     if( escapeList === null )
     {
-        escapeList = [];
+        escapeList = [
+            "current-site-button",
+            "sidebar-collapse-button",
+        ];
```

Comment:
I think we should also add these defaults if `escapeList` is not null and doesn't already contain these.

Reply:
I also felt weird doing it, but nothing else came to mind. Please explain more about what I should do here.

Reply:
Maybe something like this:

```suggestion
    if( escapeList === null )
    {
        escapeList = [];
        escapeList = [
            "current-site-button",
            "sidebar-collapse-button"
        ];
    }
    else
    {
        if( !escapeList.includes("current-site-button") )
        {
            escapeList.push( "current-site-button" );
        }
        if( !escapeList.includes("sidebar-collapse-button") )
        {
            escapeList.push( "sidebar-collapse-button" );
        }
    }
```

I'm not very up-to-date on my JS, there may be a more elegant approach to do this.

Reply:
This could still result in multiple entries for the default strings. Maybe that doesn't matter though.

Reply:
escapeList = new Set(escapeList)
escapeList.add("current-site-button").add("sidebar-collapse-button")
escapeList = [...escapeList]


You are right, I think, I will commit @bjones86 suggestion. because it will be consistent with the rest of the code